apps/modernization-ui/src/apps/page-builder/pages/BusinessRulesLibrary/Edit/EditBusinessRules.tsx

```diff
@@ -0,0 +1,203 @@
+import { Button, ButtonGroup, Form, Grid, Icon, ModalRef, ModalToggleButton } from '@trussworks/react-uswds';
+import { PagesBreadcrumb } from 'apps/page-builder/components/PagesBreadcrumb/PagesBreadcrumb';
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { useNavigate, useParams } from 'react-router-dom';
+import { UserContext } from 'user';
+import './EditBusinessRules.scss';
+// import { EditBusinessRulesFields } from './EditBusinessRulesFields';
+import { PageRuleControllerService, ViewRuleResponse } from '../../../generated';
+import DeleteBusinessRuleWarningModal from './DeleteBusinessRuleWarningModal';
+import { useAlert } from 'alert';
+
+export type FormValues = {
+    ruleFunction: string;
+    sourceIdentifier: string;
+    sourceText: string;
+    comparator?: string;
+    sourceValue: string[];
+    targetValueIdentifier?: string[];
+    targetType: string;
+    ruleDescription: string;
+    targetValueText?: string[];
+    anySourceValue?: boolean;
+};
+
+const EditBusinessRules = () => {
+    const [selectedFieldType, setSelectedFieldType] = useState('');
+    const navigate = useNavigate();
+    const { state } = useContext(UserContext);
+    const { pageId, ruleId } = useParams();
+    const form = useForm<FormValues>({
+        defaultValues: { targetType: 'SUBSECTION', anySourceValue: false },
```

Comment:
Suggestion: if you pass in the entire rule object (in the case of edit) instead of calling `PageRuleControllerService.viewRuleResponseUsingGet` within this component, you can set the defaults from this instead of setting the default values, making the API call and then manually updating the form.